app/backend/src/couchers/migrations/versions/5e89dd9ef181_adding_tokens_and_booleans_to_user_model.py

```diff
@@ -0,0 +1,55 @@
+"""adding tokens and booleans to user model
+
+Revision ID: 5e89dd9ef181
+Revises: bf12729fa8eb
+Create Date: 2021-06-14 02:27:05.166141
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "5e89dd9ef181"
+down_revision = "bf12729fa8eb"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("users", sa.Column("need_to_confirm_via_new_email", sa.Boolean(), nullable=True))
+    op.add_column("users", sa.Column("need_to_confirm_via_old_email", sa.Boolean(), nullable=True))
+    op.add_column("users", sa.Column("old_email_token", sa.String(), nullable=True))
+    op.add_column("users", sa.Column("old_email_token_created", sa.DateTime(timezone=True), nullable=True))
+    op.add_column("users", sa.Column("old_email_token_expiry", sa.DateTime(timezone=True), nullable=True))
+    # ### end Alembic commands ###
+
+    # Manually generated
+    op.create_check_constraint(
+        constraint_name="check_old_email_token_state",
+        table_name="users",
+        condition="(need_to_confirm_via_old_email IS NULL AND old_email_token IS NULL AND old_email_token_created IS NULL AND old_email_token_expiry IS NULL) OR \
+         (need_to_confirm_via_old_email IS TRUE AND old_email_token IS NOT NULL AND old_email_token_created IS NOT NULL AND old_email_token_expiry IS NOT NULL) OR \
+         (need_to_confirm_via_old_email IS FALSE AND old_email_token IS NULL AND old_email_token_created IS NULL AND old_email_token_expiry IS NULL)",
+    )
+    op.create_check_constraint(
+        constraint_name="check_new_email_token_state",
+        table_name="users",
+        condition="(need_to_confirm_via_new_email IS NULL AND new_email_token IS NULL AND new_email_token_created IS NULL AND new_email_token_expiry IS NULL) OR \
+         (need_to_confirm_via_new_email IS TRUE AND new_email_token IS NOT NULL AND new_email_token_created IS NOT NULL AND new_email_token_expiry IS NOT NULL) OR \
+         (need_to_confirm_via_new_email IS FALSE AND new_email_token IS NULL AND new_email_token_created IS NULL AND new_email_token_expiry IS NULL)",
+    )
+
```

Comment:
It might be good to set all the email verification columns to None here to make sure there's no inconsistent states.

Reply:
You mean default values?

Reply:
Yeah; basically to set all the old (before this PR) email change fields to `None`, so that if someone was in the process of changing their email it won't cause any illegal states. (They'll just have to start again, which is OK)

Reply:
Once this merges, those tokens will essentially be invalid, since I changed the name, no? New field in the model

Reply:
Not the `new_email_*` ones?